### PR TITLE
Update README, add Dockerfile

### PR DESCRIPTION
**Pull request by mor1**
Am on OSX so trying to build the site via a container threw up some missing/outdated packages. Have tried to update README to take account of them.

### CODE REVIEW

**Comment by clandmeter** [comment]
Hi Richard

Welcome to Alpine :)

Did you verify the mimetype for SVG is correctly served?
Internally we used darkhttpd for our www which I patched to support it out of the box.

**Comment by mor1** [comment]
Hi Carlo
Thanks!

I did not as this was just for testing purposes locally (Firefox displayed all the pages I clicked on ok at least :) and I was just following the instructions in the README. Does `busybox-extra httpd` not handle mimetypes correctly out of the box?

**Comment by clandmeter** [comment]
I don't think it supports SVG out of the box. If I test it locally it also doesn't show the logo.

**Comment by mor1** [comment]
Ah ok, hadn't spotted that. I'll take a look. It seems like `httpd.conf` may be my friend in this... :)

**Comment by mor1** [comment]
Ok-- that looks like it works for me now. Added `Content-type` for `.svg` and `.ico`. Logo seems to display ok anyway :)

File: Makefile [hunk 1]
```diff
@@ -79,5 +79,5 @@ DOCKER_IMAGE = alpine-mksite
 build:
 	docker build -t $(DOCKER_IMAGE) .
 test:
-	docker ps -aq --filter="ancestor=$(DOCKER_IMAGE)" | xargs docker kill
+	docker ps -aq --filter="port=8000/tcp" | xargs docker rm -f
```
Comment (ncopa) [inline Makefile]: i get: `Error response from daemon: Invalid filter 'port'`

Comment (mor1) [reply Makefile]: Hm. I guess `filter="port..."` was only added relatively recently. Will push a less fancy solution.

**Pull request closed without merging**